indra/db/__init__.py

```diff
@@ -36,6 +36,7 @@ def _compile_drop_table(element, compiler, **kwargs):
 DEFAULTS_FILE = path.join(__path__[0], 'defaults.txt')
 
 
+<<<<<<< Updated upstream
```

Comment:
There seems to be an unresolved merge conflict here that you need to resolve. This would error when trying to load the db module.